### PR TITLE
add ergodash mini

### PR DESCRIPTION
**Pull request by sfender**
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ergodash/mini/keymaps/custom/keymap.c [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```
Comment (fauxpark) [inline keyboards/ergodash/mini/keymaps/custom/keymap.c]: These two keycodes are already defined in QMK, there's no need to redefine them here :)

File: keyboards/ergodash/mini/keymaps/custom/keymap.c [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
```
Comment (fauxpark) [inline keyboards/ergodash/mini/keymaps/custom/keymap.c]: `break` after `return` is pointless, you can remove these.

File: keyboards/ergodash/mini/keymaps/custom/keymap.c (hunk 1 above)
Comment (sfender) [reply keyboards/ergodash/mini/keymaps/custom/keymap.c]: I`m so sorry to had a misstake of pull request base repository.
I close this request.

Comment (sfender) [reply keyboards/ergodash/mini/keymaps/custom/keymap.c]: I`m so sorry to had a misstake of pull request base repository.
I close this request.

**Pull request closed without merging**